### PR TITLE
chore(gatsby-transformer-screenshot): Use createContentDigest helper for file name

### PR DESCRIPTION
**Pull request by KorawitRupanya**
…-transformer-screenshot & replace peerDependency for gatsby-transformer-sqip

## Related Issues
#12973

### CODE REVIEW

File: packages/gatsby-transformer-screenshot/lambda/index.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ exports.run = async (browser, url, width, height, fullPage) => {
   }
 
   const keyBase = `${url}-(${width},${height})`
-  const digest = crypto
+  const contentDigest = createContentDigest
```
Comment (sidharthachatterjee) [inline packages/gatsby-transformer-screenshot/lambda/index.js]: We can remove the chained `.createHash(`md5`).update(keyBase).digest(`hex`)` and replace with 
`createContentDigest({ url, width, height }) `

Comment (KorawitRupanya) [reply packages/gatsby-transformer-screenshot/lambda/index.js]: Thank you 😄

File: packages/gatsby-transformer-screenshot/lambda/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const setup = require(`./starter-kit/setup`)
 
-const crypto = require(`crypto`)
+const createContentDigest = jest.fn().mockReturnValue(`contentDigest`)
```
Comment (sidharthachatterjee) [inline packages/gatsby-transformer-screenshot/lambda/index.js]:
```suggestion
const { createContentDigest } = require(`gatsby-core-utils`)
```

File: packages/gatsby-transformer-sqip/package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "debug": "^3.2.6"
   },
   "peerDependencies": {
-    "gatsby": "^2.0.0",
+    "gatsby": "^2.0.15",
```
Comment (sidharthachatterjee) [inline packages/gatsby-transformer-sqip/package.json]:
```suggestion
    "gatsby": "^2.0.0",
```

File: packages/gatsby-transformer-sqip/package.json [hunk 4]
```diff
@@ -24,7 +24,7 @@
...
     "gatsby-source-contentful": "^2.0.0",
```
Comment (sidharthachatterjee) [inline packages/gatsby-transformer-sqip/package.json]: Let's add `gatsby-core-utils` as a dependency. You can do this by running `yarn add gatsby-core-utils` inside this folder.

Comment (KorawitRupanya) [reply packages/gatsby-transformer-sqip/package.json]: I keep getting this error `error https://github.com/okonet/listr-update-renderer/tarball/upgrade-log-update: Integrity check failed for "listr-update-renderer" (computed integrity doesn't match our records, got "sha1-Bgc/qTFmJ3YHp4FPTh+DlgCBQUw=")
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
` What can I do?

**Comment by KorawitRupanya** [comment]
Thank you so much and really sorry if I made you waste your time 😭

**Pull request merged**